### PR TITLE
new tear gas sprite

### PR DESCRIPTION
**Pull request by LentilMan**
## About the PR

tear gas isnt in the same perspective as other grenades
## Why / Balance

kinda bothered me, so i decided to make a new sprite in GIMP that matches the perspective as the other grenades (if anything is wrong im sorry,)
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
image (1)

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by LentilMan** [comment]
hopefully i didnt leave any mistakes

**Comment by metalgearsloth** [comment]
At least center it fam.

File: finalized tear gas sprite.xcf
Comment (LentilMan) [inline finalized tear gas sprite.xcf:1]: This file is kinda centered, and has no other mistakes

**Comment by EmoGarbage404** [comment]
Needs in game screenshots side by side with other grenades.

**Pull request closed without merging**